### PR TITLE
Volunteer apps fixes

### PR DESCRIPTION
**Pull request by alexandrawang1**
Factors out base path for volunteer application links 
Allows admins to toggle applications by setting active to false

### CODE REVIEW

File: views/pages/get-involved/volunteer-application.pug [hunk 1]
```diff
@@ -39,7 +39,7 @@ block body
           label(for="coop-false") No
         
         label(for="interest" class="required") Please explain why you are interested in this volunteer role.
-        textarea(id="interest" class="text-input" name="interest" required maxlength="6000")
+        textarea(id="interest" class="text-input" name="interest" required maxlength="6000" pattern=".*[^ ].*" oninvalid="this.setCustomValidity('Please enter your answer')" onchange="try{setCustomValidity('')}catch(e){}" oninput="setCustomValidity(' ')")
```
Comment (alexandrawang1) [inline views/pages/get-involved/volunteer-application.pug:42]: I thought I already did this 🧍 But it just adds custom error messages for invalid (empty) forms

File: server/data-base/get-involved/volunteer.json [hunk 2]
```diff
@@ -2,39 +2,46 @@
   "version": 1,
   "title": "Volunteer with MathSoc!",
   "subtext": "Volunteering with MathSoc is a great way to meet other people in your faculty. While some positions are time sensitive we welcome applications at anytime. If you have questions about a role we recommend sending the corresponding VP or club an email. If a position that you are interested in is not currently open and you wish to receive a notification when it opens please email the VPC at vpc@mathsoc.uwaterloo.ca about what positions you are interested in and ensure to check this page regularly. We look forward to meeting you!",
+  "path": "get-involved/volunteer-application",
```
Comment (some-molasses) [inline server/data-base/get-involved/volunteer.json:5]: did this get missed in the `schemas.ts` update? 👀

File: server/types/schemas.ts [hunk 3]
```diff
@@ -26,6 +26,7 @@ const VolunteerDataSchema = z
                 title: z.string(),
                 descriptionMarkdown: z.string(),
                 link: z.string(),
+                active: z.boolean(),
```
Comment (some-molasses) [inline server/types/schemas.ts:29]: can you take a look at `migrations.ts` and create a quick migration to automatically change the data on the staging site when this merges?

**Review by some-molasses — APPROVED** [review]
approved so long as the schema is right (see comment)!

**Review by some-molasses — APPROVED** [review]
🎉

**Pull request merged**